wikimedia/wikipedia-ios
# [ALT TEXT] Add feedback alerts

**Phabricator:** https://phabricator.wikimedia.org/T370360

### Notes
* This PR adds the feedback alerts with post-edit surveys

### Test Steps
1. Go through flow B to the end
2. After publishing alt text, you should get a Yes/No survey
3. Note that the alert has the buutons side-by-side. This is the default with two options for this type of alert.
4. Go through flow C to the end.
5. You should get a survey with three options. After clicking one option, the same Yes/No survey above displays.
6. Check the event logging and translation
7. Also look out for the toasts!
